src/app/pages/overlays/basic-bottom-sheet/bottom-sheet/bottom-sheet.scss

```diff
@@ -0,0 +1,7 @@
+.mat-bottom-sheet-container {
+    padding: 0;
+}
+
+mat-nav-list {
+    padding-top: 0 !important;
```

Comment:
does this need to be important?

Reply:
Done. I was able to remove this by changing the selector to a class instead of the element. For some reason, I needed important when using the element selector?